### PR TITLE
New CRAN/R packages Spice

### PR DESCRIPTION
**Pull request by pjsix**
https://duck.co/ia/view/cran

To do: 

1. Get rid of "\<U+000a\>" (Unicode code for newline) in output. E.g. "r sm".
2. Do not truncate fields (unless they're really long). E.g. the Author field of "r sm".
3. I put the package description in the subtitle field, as it doesn't truncate. Looks fine to me, but is that how it should be done?

### CODE REVIEW

**Comment by daxtheduck** [comment]
@forogo Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: lib/DDG/Spice/CRAN.pm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package DDG::Spice::CRAN;
+
+# ABSTRACT: Search for information about R packages in the CRAN repository (using METACRAN's CRAN database)
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
```
Comment (GuiltyDolphin) [inline lib/DDG/Spice/CRAN.pm]: Remember to get rid of all the template comments!

**Comment by tagawa** [comment]
@forogo, thanks again for creating the CRAN/R packages IA! You’ve been doing a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in statistics.

When you're ready to move on, please:

- Create a new Instant Answer for searches you do related to statistics

OR

- Test live statistics Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Comment by moollaza** [comment]
This Pull Request is being closed due to inactivity. If you're still interested in finishing, please let us know and we'll gladly re-open it for you!

**Pull request closed without merging**